pkgs/data/fonts/helvetica-neue-lt-std/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation {
+  name = "helvetica-neue-lt-std";
```

Comment:
Need to have a version here. Otherwise, Nix will interpret the version to be "std". Could be a date, for example `name = "helvetica-neue-lt-std-20130607`.